### PR TITLE
fix metering for state new

### PR DESCRIPTION
**Pull request by LenaHoinkis**
### Description

The test case "success_when_state_new" failed with the error message: "cannot observe StateItem if StartedAt is not set".
In State new, we need to use the creation timestamp to get the start time.

### Submitter checklist

- [ ] Change has been tested (on a back-end cluster)
- [ ] (If applicable) Jira backlog item ID added to the PR title and the [changelog.yaml] entry
- [ ] [changelog.yaml] entry with upgrade notes is prepared and appropriate for the audience affected by the change (users or developer, depending on the change)
- [ ] Semantic version diffed against [last release][releases] and updated accordingly. In this project the version has to be maintained here:
    - [/charts/steward/Chart.yaml](https://github.com/SAP/stewardci-core/blob/master/charts/steward/Chart.yaml) (`version` and `appVersion`)

In case dependencies have been updated:
- [ ] Links to external changelogs, since the last release of our component, added to the [changelog.yaml] entry (description).
- [ ] Changelogs read thoroughly, potential impact described, upgrade notes prepared (if necessary)
- [ ] Check if dependency updates affect our semantic version increment.

### Reviewer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There is at least 1 approval for the pull request and no outstanding requests for change
- [ ] All voter checks have passed
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] The Pull Request title is understandable and reflects the changes well
- [ ] The Pull Request description is understandable and well documented
- [ ] [changelog.yaml] entry for this Pull Request has been added
    - [ ] Changelog entry contains all required information
    - [ ] 'Upgrade notes' are documented in changelog.yaml (if required)

[changelog.yaml]: https://github.com/SAP/stewardci-core/changelog.yaml
[releases]: https://github.com/SAP/stewardci-core/releases

### CODE REVIEW

**Comment by rinckm** [comment]
Please add changelog

File: changelog.yaml [hunk 1]
```diff
@@ -52,6 +52,15 @@
   date: TBD
   changes:
 
+    - type: bug
+       impact: patch
```
Comment (rinckm) [inline changelog.yaml:56]: Looks like an error in the indentation.

**Review by rinckm — CHANGES_REQUESTED** [review]
See comments inline

File: changelog.yaml [hunk 2]
```diff
@@ -52,6 +52,15 @@
   date: TBD
   changes:
 
+    - type: bug
+      impact: patch
+      title: Fix metering current durations in state new
+      description: |-
+        Observing the current duration of the pipelineruns failed in state new.
+        It could not determine the start timestamp and returned the error:
+        "cannot observe StateItem if StartedAt is not set"
```
Comment (anbrsap) [inline changelog.yaml:61]:
```suggestion
        Measuring ongoing state duration failed for pipeline runs in state `new`
        with error message:
        
        ```
        cannot observe StateItem if StartedAt is not set
        ```
```

File: changelog.yaml [hunk 3]
```diff
@@ -52,6 +52,15 @@
   date: TBD
   changes:
 
+    - type: bug
+      impact: patch
+      title: Fix metering current durations in state new
```
Comment (anbrsap) [inline changelog.yaml:57]:
```suggestion
      title: Fix measuring ongoing state durations
```

File: changelog.yaml (hunk 1 above)
Comment (LenaHoinkis) [reply changelog.yaml]: repaired

**Review by rinckm — APPROVED** [review]

**Pull request merged**